src/components/MessagesList/MessagesGroup/Message/Message.vue

```diff
@@ -157,6 +157,12 @@ the main body of the message as well as a quote.
 								:href="linkToFile">
 								{{ t('spreed', 'Go to file') }}
 							</ActionLink>
+							<ActionLink
+								icon="icon-share"
```

Comment:
Should use Material design icon `Share` instead

Reply:
How to use it within an ActionLink componente?

Reply:
Well first don't use ActionLink when you want a buttom instead of a link, and then the docs have some samples:
https://nextcloud-vue-components.netlify.app/#/Components/Actions?id=actionbutton

```vue
		<ActionButton>
			<HandLeft
				slot="icon"
				:size="24"
				decorative
				title="" />
			Raise left hand
		</ActionButton>
```